src/components/MultipleValueTextInput.tsx

```diff
@@ -84,6 +104,33 @@ function MultipleValueTextInput({
 			handleItemAdd(e.target.value);
 		}
 	};
+
+	const splitMulti = (str: string) => {
+		const tempChar = delimiters[0] // We can use the first token as a temporary join character
+		let result: string = str
+		for (let i = 1; i < delimiters.length; i+=1) {
+			result = result.split(delimiters[i]).join(tempChar) // Handle scenarios where pasted text has more than one submitKeys in it
+		}
+		return result.split(tempChar)
+	}
+
+	const handlePaste = (e: React.ClipboardEvent<HTMLInputElement>) => {
+		const pastedText = e.clipboardData.getData('text/plain')
+		let areSubmitKeysPresent = false
```

Comment:
Another fun bit of Javascript array syntax to play with - lines 119-124 could be replaced with:

```
const areSubmitKeysPresent = delimiters.some((d) => pastedText.includes(d));
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some